core/common/src/main/java/alluxio/util/webui/UIMetric.java

```diff
@@ -0,0 +1,39 @@
+package alluxio.util.webui;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The type Ui metric.
+ */
+public class UIMetric {
+  private long mValue;
+
+  /**
+   * Instantiates a new Ui metric.
+   *
+   * @param value the value
+   */
+  @JsonCreator
+  public UIMetric(@JsonProperty("value") long value) {
+    mValue = value;
+  }
+
+  /**
+   * Sets value.
+   *
+   * @param value the value
+   */
+  public void setValue(long value) {
+    mValue = value;
+  }
+
+  /**
+   * Gets value.
+   *
+   * @return the value
+   */
```

Comment:
Overall comment, for getters/setters we usually do not provide a javadoc comment, just the `@param` and `@return`

Reply:
got it, made some changes